transactions.php

```diff
@@ -225,13 +337,15 @@ function getTransaction($transaction_id) {
                     transactions.name AS transaction_name,
                     transactions.date AS transaction_date,
                     transactions.description AS transaction_description,
+                    group_transactions.group_id as group_id,
                     CASE WHEN COUNT(tp.user_id) = SUM(tp.has_approved)
                         THEN 1
                         ELSE 0
                     END AS is_approved
       
             FROM transactions
             JOIN transaction_participants tp ON tp.transaction_id = transactions.transaction_id
+            JOIN group_transactions ON group_transactions.transaction_id = transactions.transaction_id
```

Comment:
again, should be a left join, but will that return NULL in the SELECT for group_transactions.group_id?